src/loki/status_action.cc

```diff
@@ -1,8 +1,23 @@
 #include "loki/worker.h"
+#include "proto/status.pb.h"
 
 namespace valhalla {
 namespace loki {
-void loki_worker_t::status(Api&) const {
+void loki_worker_t::status(Api& request) const {
+  baldr::graph_tile_ptr tile = nullptr;
+  for (const auto& tile_id : reader->GetTileSet(2)) {
```

Comment:
i wouldnt limit to level 2 i dont think it adds anything extra it still doing a directory scan in tile_dir mode. there is the potential for that to be a huge disk crawling operation which isnt exactly the best idea. personally i think we should cache the answer to this, which reminds me of the connectivity map. maybe it makes sense to simply return information from that? i know you can disable the connectivity map but its only useful to do that when you are constantly changing what tiles you have in cache. speaking of such senarios, there is one pitfall here that i dont think we can overcome which is if you configure your graphreader to only hit http to get tiles. also there is no need to do the call to `DoesTileExist` the `GetTileSet` call ensures it exists if it returns it.